extension/src/shared/ui-components/StyledButton.tsx

```diff
@@ -0,0 +1,13 @@
+import { Button, styled } from "@mui/material";
+
+export const StyledButton = styled(Button)(({ theme, disabled }) => ({
```

Comment:
This should be placed at `/ui-components/MyData/StyledButton.tsx`. Because I think this is depending on the UI of MyData.